privcount/data_collector.py

```diff
@@ -714,52 +782,52 @@ def _handle_stream_event(self, items):
         lifetime = end-start
 
+        self.secure_counters.increment('ExitStreamCount', SINGLE_BIN)
+        self.secure_counters.increment('ExitStreamByteCount', SINGLE_BIN, totalbw)
         self.secure_counters.increment('ExitStreamOutboundByteCount', writebw)
         self.secure_counters.increment('ExitStreamInboundByteCount', readbw)
         self.secure_counters.increment('ExitStreamByteRatio', ratio)
+        self.secure_counters.increment('ExitStreamLifeTime', lifetime)
 
         if stream_class == 'web':
-            self.secure_counters.increment('ExitWebStreamCount', 1)
-            self.secure_counters.increment('ExitWebStreamByteCount', 1, totalbw)
+            self.secure_counters.increment('ExitWebStreamCount', SINGLE_BIN)
+            self.secure_counters.increment('ExitWebStreamByteCount', SINGLE_BIN, totalbw)
             self.secure_counters.increment('ExitWebStreamOutboundByteCount', writebw)
             self.secure_counters.increment('ExitWebStreamInboundByteCount', readbw)
             self.secure_counters.increment('ExitWebStreamByteRatio', ratio)
             self.secure_counters.increment('ExitWebStreamLifeTime', lifetime)
         elif stream_class == 'interactive':
-            self.secure_counters.increment('ExitInteractiveStreamCount', 1)
-            self.secure_counters.increment('ExitInteractiveStreamByteCount', 1, totalbw)
+            self.secure_counters.increment('ExitInteractiveStreamCount', SINGLE_BIN)
+            self.secure_counters.increment('ExitInteractiveStreamByteCount', SINGLE_BIN, totalbw)
             self.secure_counters.increment('ExitInteractiveStreamOutboundByteCount', writebw)
             self.secure_counters.increment('ExitInteractiveStreamInboundByteCount', readbw)
             self.secure_counters.increment('ExitInteractiveStreamByteRatio', ratio)
             self.secure_counters.increment('ExitInteractiveStreamLifeTime', lifetime)
         elif stream_class == 'p2p':
-            self.secure_counters.increment('ExitP2PStreamCount', 1)
-            self.secure_counters.increment('ExitP2PStreamByteCount', 1, totalbw)
+            self.secure_counters.increment('ExitP2PStreamCount', SINGLE_BIN)
+            self.secure_counters.increment('ExitP2PStreamByteCount', SINGLE_BIN, totalbw)
             self.secure_counters.increment('ExitP2PStreamOutboundByteCount', writebw)
             self.secure_counters.increment('ExitP2PStreamInboundByteCount', readbw)
             self.secure_counters.increment('ExitP2PStreamByteRatio', ratio)
             self.secure_counters.increment('ExitP2PStreamLifeTime', lifetime)
         elif stream_class == 'other':
-            self.secure_counters.increment('ExitOtherPortStreamCount', 1)
-            self.secure_counters.increment('ExitOtherPortStreamByteCount', 1, totalbw)
+            self.secure_counters.increment('ExitOtherPortStreamCount', SINGLE_BIN)
+            self.secure_counters.increment('ExitOtherPortStreamByteCount', SINGLE_BIN, totalbw)
```

Comment:
This actually makes the code really hard to read:
https://github.com/teor2345/privcount/commit/23d5fb4c7189eaae74c9aad30913b49fb4e2a9c0

Instead, I suggest we always use the 3-argument form, with shorter keywords, on separate lines. See 4eeae10 for how this looks, and 87bcc44 for a working version.